src/assets/data/team/team-members.json

```diff
@@ -0,0 +1,87 @@
+[
+  { "name": "Jennifer Dutra", "title": "President", "imgRef": "jen-dutra.jpg" },
+  {
+    "name": "Daniel Segobiano",
+    "title": "President",
+    "imgRef": "daniel-segobiano.jpg"
+  },
+  {
+    "name": "Kyle O'Brien",
+    "title": "Lead Engineer",
```

Comment:
Change title to "Engineering Director"

Reply:
You got it.

Reply:
Also in reference to the wide cards. Can I get your screen resolution as a reference for a good baseline? I have a good idea of what values to use now.

Reply:
to keep the cards from overextending, just add a max-width. don't overcomplicate with too many media breakpoints!

Reply:
Try Eric's suggestion